doc/source/towncrier.md

```diff
@@ -0,0 +1,20 @@
+If automated deployment of release notes fails you can follow the manual
+instructions in this document.
+
+- Identify if the Notes artifact is currently available for the failed
+“release” workflow; If that is the case, you can manually download the
+artifacts and extract the release file, you can then follow the process of
+creating a release on Github and submit the release notes as its contents;
+
+- If the Notes artifacts are currently not available because the CI action
+failed or is not available you can run the following steps on your local
+machine.
+
+```
+cd <path/to/scikit-image/repository>
+pip install -r requirements/_release_docs.txt
+python tools/aggregate_contributors <path/to/scikit-image>
```

Comment:
```suggestion
python tools/aggregate_contributors .
```
Using `<path/to/scikit-image>` her and `<path/to/scikit-image/repository>` above can potentially confuse readers into thinking these are two different paths

we are already in the repository path due to the command above, so I just switched to `.` here